src/scripts/weapons/zp_weapon_shotgun.sma

```diff
@@ -43,7 +45,10 @@ public plugin_init() {
 }
 
 public @Weapon_PrimaryAttack(this) {
-  if (CW_DefaultShotgunShot(this, 6.0, 1.0, 0.5, Float:{0.0975, 0.0975, 0.0975}, 16)) {
+  static Float:vecSpread[3];
+  UTIL_CalculateWeaponSpread(this, Float:{0.0975, 0.0975, 0.0975}, 1.5, 0.95, 2.0, vecSpread);
```

Comment:
`Float:{0.0975, 0.0975, 0.0975}` need to define to const like `Float:VECTOR_CONE_2DEGREES`

Reply:
This value is something between VECTOR_CONE_10DEGREES and VECTOR_CONE_15DEGREES, let it be so.

Reply:
need to calculate 12deg by this formula 
https://github.com/johannesfknudsen/SourceMod/blob/master/mp/src/game/shared/basecombatweapon_shared.h#L94-L116